<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FDL - PyQt5 Downloader</string>
  </property>
  <widget class="QWidget" name="fdldialog">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="mTaskMenu">
    <property name="title">
     <string>Task</string>
    </property>
    <addaction name="mAddNewTaskAction"/>
    <addaction name="mExitAction"/>
   </widget>
   <widget class="QMenu" name="mFileMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="mStopDownloadAction"/>
   </widget>
   <widget class="QMenu" name="mDownloadMenu">
    <property name="title">
     <string>Download</string>
    </property>
    <addaction name="mOptionalAction"/>
   </widget>
   <widget class="QMenu" name="mHelpMenu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="mAboutAction"/>
    <addaction name="mUpdateAction"/>
   </widget>
   <widget class="QMenu" name="mSettingMenu">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuUI_Language">
     <property name="title">
      <string>UI Language</string>
     </property>
     <addaction name="mEnglishAction"/>
     <addaction name="mChineseAction"/>
    </widget>
    <addaction name="menuUI_Language"/>
   </widget>
   <addaction name="mTaskMenu"/>
   <addaction name="mFileMenu"/>
   <addaction name="mDownloadMenu"/>
   <addaction name="mSettingMenu"/>
   <addaction name="mHelpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string notr="true">toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="mAddNewTaskAction">
   <property name="text">
    <string>Add New Task(N)</string>
   </property>
   <property name="toolTip">
    <string>add new task</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+N</string>
   </property>
  </action>
  <action name="mExitAction">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>exit</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Q</string>
   </property>
  </action>
  <action name="mStopDownloadAction">
   <property name="text">
    <string>Stop Download</string>
   </property>
   <property name="toolTip">
    <string>stop download</string>
   </property>
  </action>
  <action name="mOptionalAction">
   <property name="text">
    <string>Optional</string>
   </property>
   <property name="toolTip">
    <string>optional</string>
   </property>
  </action>
  <action name="mAboutAction">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>about</string>
   </property>
  </action>
  <action name="mUpdateAction">
   <property name="text">
    <string>Update</string>
   </property>
   <property name="toolTip">
    <string>update</string>
   </property>
  </action>
  <action name="mEnglishAction">
   <property name="text">
    <string notr="true">English</string>
   </property>
  </action>
  <action name="mChineseAction">
   <property name="text">
    <string notr="true">中文简体</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
